src/test/java/com/google/sps/ServletUtilsTest.java

```diff
@@ -0,0 +1,131 @@
+package com.google.sps;
```

Comment:
I left comments on the ServletUtilsTest in the other PR. Please check those if you're merging this PR first